posthog/api/test/test_exports.py

```diff
@@ -594,6 +594,44 @@ def test_retrieve_shows_stuck_export_as_failed_in_response(self) -> None:
         stuck_export.refresh_from_db()
         self.assertIsNone(stuck_export.exception)
 
+    def test_can_filter_exports_by_format(self) -> None:
+        png_export = ExportedAsset.objects.create(
+            team=self.team, dashboard_id=self.dashboard.id, export_format="image/png", created_by=self.user
+        )
+        csv_export = ExportedAsset.objects.create(
+            team=self.team, insight_id=self.insight.id, export_format="text/csv", created_by=self.user
+        )
+
+        response = self.client.get(f"/api/projects/{self.team.id}/exports?export_format=image/png")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        results = response.json()["results"]
+
+        png_export_ids = {png_export.id, self.exported_asset.id}
+        returned_ids = {result["id"] for result in results}
+        assert returned_ids == png_export_ids
+
+        for result in results:
+            assert result["export_format"] == "image/png"
+
+        response = self.client.get(f"/api/projects/{self.team.id}/exports?export_format=text/csv")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        results = response.json()["results"]
+
+        assert len(results) == 1
+        assert results[0]["id"] == csv_export.id
+        assert results[0]["export_format"] == "text/csv"
+
+        # Unsupported format should return all exports since filter is ignored
+        response = self.client.get(f"/api/projects/{self.team.id}/exports?export_format=image/jpeg")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        results = response.json()["results"]
+        assert len(results) == 3
```

Comment:
evergreen prompt to split up multiple assertions like this into a parameterized test